### ISSUE TITLE
Programming guides on with multiple 'argscope'

### ISSUE THREAD

**Issue by MiZhangWhuer** [tensorpack/tensorpack#1147]
Thanks for sharing this impressive work. In the vgg16.py example, I notice that the argscope is defined twice in the get_logits function. What confused me most is why Conv2D is used the second time in argscope? Additionally, is there any guide for using multiple argscope?  I can find the guide for using one argscope in the tutorial part, but I can't find any guides on using argscope multiple times. Would you please provide more guides on how to use multiple argscope?  

def get_logits(self, image):   
     with argscope(Conv2D, kernel_initializer=tf.variance_scaling_initializer(scale=2.)),   
                argscope([Conv2D, MaxPooling, BatchNorm], data_format='channels_first'):

**Comment by ppwwyyxx** [comment]
Added in the tutorial now: https://tensorpack.readthedocs.io/tutorial/symbolic.html
